website/docs/content/voice-and-tone/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Voice and Tone
```

Comment:
**todo:** titles and headings use sentence case ([reference](https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/3342009417/Writing+Style#Capitalization))
```suggestion
title: Voice and tone
```

Reply:
^^ same as above, these were added back incorrectly. Its deleted again.